packages/framework-integration-tests/integration/helper/app-helper.ts

```diff
@@ -12,6 +13,7 @@ export async function getProviderTestHelper(): Promise<ProviderTestHelper> {
   const provider = process.env.TESTED_PROVIDER
   const providerHelpers: Record<string, () => Promise<ProviderTestHelper>> = {
     AWS: () => AWSTestHelper.build(applicationName()),
+    LOCAL: () => LocalTestHelper.build(applicationName()),
```

Comment:
The integration tests can receive `LOCAL` as a value to run the suite with the Local Provider

Reply:
💥